testdata/serviceconfig.yml

```diff
@@ -15,16 +15,16 @@ rules:
         # we want to increment this counter by 1 for each unique (source, target, service, method, response_code) tuple
         value: "1"
         labels:
-          source: source.name
-          target: target.name
+          source: source.name | "unknown"
+          target: target.name | "unknown"
           service: api.name | "unknown"
           method: api.method | "unknown"
-          response_code: response.http.code
+          response_code: response.http.code | "unknown"
       - descriptor_name:  request_latency
-        value: response.latency
+        value: response.latency | "unknown"
```

Comment:
I mean from a syntactic point of view, I don't know how I write a duration literal that the expression evaluator will understand. Deciding how that zero value is handled is a totally separate issue that we can punt on till after the demo.